scrapers/az/bills.py

```diff
@@ -353,34 +353,42 @@ def scrape_votes(self, bill, page):
     def scrape(self, chamber=None, session=None):
         session_id = session_metadata.session_id_meta_data[session]
 
-        # Get the bills page to start the session
-        req = self.get("https://www.azleg.gov/bills/", timeout=80)
+        init_req = self.get("https://www.azleg.gov/bills/", timeout=80)
+        cookies = init_req.cookies
 
         session_form_url = "https://www.azleg.gov/azlegwp/setsession.php"
         form = {"sessionID": session_id}
         headers = {
-            "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9",
+            "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8",
             "Content-Type": "application/x-www-form-urlencoded; charset=utf-8",
             "Referer": "https://www.azleg.gov/azlegwp/",
-            "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36",
+            "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36",
```

Comment:
this is a little odd, I wonder if we'll have to periodically rotate the user agent